search_commands.go

```diff
@@ -261,7 +262,8 @@ type FTAggregateOptions struct {
 	WithCursor        bool
 	WithCursorOptions *FTAggregateWithCursor
 	Params            map[string]interface{}
-	DialectVersion    int
+	// Dialect 1,3 and 4 are deprecated since redis 8.0
```

Comment:
I assume we still would like to keep the DialectVersion at least till Redis 10, then we can drop it. Let's deprecate this around Redis 9 (add a task somewhere to remind ourselves).